test/bats/plugin-test.bats

```diff
@@ -30,7 +30,6 @@ SLEEP_TIME=1
     teardown() {
         echo "cleaning up"
         wait_for_process ${WAIT_TIME} ${SLEEP_TIME} 'kubectl delete pod cosign-demo-keyless --namespace default --force --ignore-not-found=true'
-        wait_for_process ${WAIT_TIME} ${SLEEP_TIME} 'kubectl delete verifiers.config.ratify.deislabs.io/verifier-cosign --namespace default --ignore-not-found=true'
```

Comment:
wondering why this was necessary? Is this related to the current build failures we are seeing?

Reply:
right, it's for resolving the issue I just saw, the error saied cosign-verifier is not found while replacing it, remove this line makes the build pass.